recipes/nasm/all/conanfile.py

```diff
@@ -105,3 +105,8 @@ def package(self):
     def package_info(self):
         self.cpp_info.libdirs = []
         self.cpp_info.includedirs = []
+
+        # remove later once all recipes upgraded to conan v2
+        bin_path = os.path.join(self.package_folder, "bin")
+        self.output.info(f"Appending PATH environment variable: {bin_path}")
+        self.env_info.PATH.append(bin_path)
```

Comment:
```suggestion
        self.buildenv_info.prepend_path("PATH", bin_path)
```

If I understand correctly, this is the right way to go.

Reply:
I used env_info which worked locally.  I'll test buildenv_info.
There was another PR recently where this (or similar) was discussed... I'll see if I can find it.

Reply:
No, that didn't work.

Reply:
https://github.com/ericLemanissier/conan-center-index/blob/f385a2ed6443ec04d29225a4eae75a052141f4c5/recipes/jom/all/conanfile.py#L50
Recent example - more than just one thing.

Reply:
> If I understand correctly, this is the right way to go.

No, it's not for PATH. It's a first class citizen.